java/piranha/src/test/java/com/uber/piranha/EnumConstantTest.java

```diff
@@ -33,6 +33,464 @@
 public class EnumConstantTest {
   @Rule public TemporaryFolder temporaryFolder = new TemporaryFolder();
 
+  /**
+   * Test for cleaning up the first enum constant in a list of enum constants.
+   *
+   * <p>See <a href="https://github.com/uber/piranha/issues/143">Issue #143</a>
+   */
+  @Test
+  public void cleanUpFirstEnumTest() {
+    ErrorProneFlags.Builder b = ErrorProneFlags.builder();
+    b.putFlag("Piranha:FlagName", "STALE_FLAG");
+    b.putFlag("Piranha:IsTreated", "false");
+    b.putFlag("Piranha:Config", "config/properties.json");
+
+    BugCheckerRefactoringTestHelper bcr =
+        BugCheckerRefactoringTestHelper.newInstance(new XPFlagCleaner(b.build()), getClass());
+
+    bcr = bcr.setArgs("-d", temporaryFolder.getRoot().getAbsolutePath());
+
+    bcr.addInputLines(
+            "TestExperimentName.java",
+            "package com.uber.piranha;",
+            "public enum TestExperimentName {",
+            " STALE_FLAG,",
+            " OTHER_FLAG_1,",
+            " OTHER_FLAG_2;",
+            "}")
+        .addOutputLines(
+            "TestExperimentName.java",
+            "package com.uber.piranha;",
+            "public enum TestExperimentName {",
+            " OTHER_FLAG_1,",
+            " OTHER_FLAG_2;",
+            "}")
+        .doTest();
+  }
+
+  /**
+   * Test for cleaning up an enum in the middle of the list of enum constants, matching on the enum
+   * constant.
+   *
+   * <p>See <a href="https://github.com/uber/piranha/issues/143">Issue #143</a>
+   */
+  @Test
+  public void cleanUpMiddleEnumTest() {
```

Comment:
Isn't there a missing `cleanUpLastEnumTest`? Particularly since the following test changes things around by passing arguments to the enum constants, so it isn't testing equivalent code.

Reply:
Hmm let me see if I can reorganize the tests a bit for clarity and have a more consistent naming scheme. cleanUpLastEnumTest would be equivalent to cleanUpLastEnumTestWithUnneededTrailingSemicolon although it has 2 constants instead of 3.